### PR TITLE
Intro page redesign

### PR DESCRIPTION
**Pull request by oscarssanchez**
Hi @tott ,

Could you please review this PR? 

This one redesigns the intro page that is displayed the first time a user clicks on the elasticpress plugin icon in the sidebar of the wp-admin area.

Before: 

epbefore

Now: 

epafter

Thanks!

### CODE REVIEW

File: includes/intro-page.php [hunk 1]
```diff
@@ -20,14 +20,41 @@
...
+			<p class="ep-copy-text"><?php esc_html_e( 'You\'ve taken your first step into a faster and more flexible search and query engine for WordPress', 'elasticpress' ); ?></p>
+		</div>
+		<div class="intro-box">
+			<div class="circle white-circle">
+				<?php esc_html_e( 'Step', 'elasticpress' ); ?><p>2</p>
+			</div>
+			<h2><?php esc_html_e( 'Set up hosting', 'elasticpress' ); ?></h2>
+			<p class="ep-copy-text">
+				<?php echo sprintf( __( 'The next step is to make sure you have a working Elasticsearch server. We recommend creating an %s account or if you want you can set up your own hosting', 'elasticpress' ),
+					'<a href="https://www.elasticpress.io/">ElasticPress.io</a>'
```
Comment (tott) [inline includes/intro-page.php]: I would link this to blank target so we're not taking people out of the workflow

Comment (ivanlopez) [reply includes/intro-page.php]: @tott it might be worth adding a UTM code so you can track traffic from this click.

**Comment by tlovett1** [comment]
What happens if `EP_HOST` is defined already?

File: includes/intro-page.php (hunk 1 above)
Comment (oscarssanchez) [reply includes/intro-page.php]: Hi @ivanlopez, Can we add the code even if the URL is set to blank ?

**Comment by oscarssanchez** [comment]
Hi @tott ,

I just added the feedback into an interactive process to configure ElasticPress. Could you pleas review? 

We are starting always on Step 2: 

step2

After setting up a host we end on Step 3 if the user visits again the elasticpress page:

step3

After a Sync, we end up on these step:

setupc

Mobile version:
setupcm

I'm wondering if this notice is still pertinent to show or should we redirect users to the intro page again after they set up their host successfully:

notice2

**Comment by oscarssanchez** [comment]
Nevermind I think I got this wrong, I'll redo this today and update

**Comment by oscarssanchez** [comment]
Updated this PR @tott , @brandwaffle, Can you give me your thoughts please?

**Comment by allan23** [comment]
@oscarssanchez  is this already in 3.0?

CC: @tlovett1

**Comment by oscarssanchez** [comment]
Looks like it is @allan23 https://github.com/10up/ElasticPress/pull/1253

**Pull request closed without merging**